### ISSUE TITLE
Add environment variables to configure TT-RSS and optional volume

### ISSUE THREAD

**Issue by Neraud** [linuxserver/docker-tt-rss#17]
Currently, TT-RSS is stored in the config volume. 
It is configured by creating a `config.php` file in this volume (either manually, or through the installer web ui)

It would be great to have the option to use environment variables to inject the configuration in the container.
In this case, I'm guessing we would need to remove the config volume definition.

We could have 2 different ways to run the container :

1. The current one, with the config volume

```
-v <path to data>:/config
```

2. Via environment variables
Basically, we could generate the `config.php` from the sample [config.php-dist](https://git.tt-rss.org/fox/tt-rss/src/master/config.php-dist) and inject all the possibe settings from there.
```
(no volume on /config)
-e DB_TYPE=pgsql or mysql
-e DB_HOST=localhost
-e DB_USER=fox
-e DB_NAME=fox
-e DB_PASS=XXXXXX
-e DB_PORT=5432 or 3306
-e SELF_URL_PATH=http://example.org/tt-rss
-e REG_NOTIFY_ADDRESS=user@your.domain.dom
...
```

**Comment by nkay08** [comment]
I agree, this seems very reasonable. 
It is also useful if running the database in another container whose hostname is not known before creating the container.

**Comment by thelamer** [comment]
Please reference this PR for updates: 
https://github.com/linuxserver/docker-tt-rss/pull/24
